service/controller/v2/resources/cleansing/resource.go

```diff
@@ -0,0 +1,16 @@
+package cleansing
```

Comment:
What is this about? I get the rename but I don't follow this.

Reply:
sorry, my bad, somehow it goes with initial commit.